src/github/SyftGithubAction.ts

```diff
@@ -197,10 +197,10 @@ export async function uploadSbomArtifact(contents: string): Promise<void> {
  */
 function getBooleanInput(name: string, defaultValue: boolean): boolean {
   const val = core.getInput(name);
-  if (val === "") {
+  if (val === undefined || val === "") {
```

Comment:
Is `null` a possible value?

Reply:
`null` is not really ever going to be set here. As a Github action, it's always returning empty string for unset properties. I was just lazy with tests and left properties undefined in the mocks. I suppose I could instead return `""`. This just seemed like a logical check to add.